src/createAsyncThunk.ts

```diff
@@ -157,7 +157,7 @@ export function createAsyncThunk<
       const aborted = !!error && error.name === 'AbortError'
       return {
         payload,
-        error: miniSerializeError(error) || { message: 'Rejected' },
+        error: payload ? { message: 'Rejected' } : miniSerializeError(error),
```

Comment:
I'd suggest this change - otherwise it would be possible to call the `rejected` actionCreator by hand with no payload and `null` as error.

```suggestion
        error: miniSerializeError(error || 'Rejected'),
```

Reply:
Good point!